command/agent/config/test-fixtures/bad-config-env-templates-invalid-signal.hcl

```diff
@@ -0,0 +1,26 @@
+auto_auth {
+
+  method {
+    type = "token_file"
+
+    config {
+      token_file_path = "$HOME/.vault-token"
```

Comment:
Nit: having examples with environment variables like this in our config file test fixtures might imply that this expands, but it doesn't, so I think we should avoid it.

Maybe instead, use
`/home/username/.vault-token` or similar?